gophersnake.py

```diff
@@ -619,7 +620,8 @@ def copy_to_clipboard():
 # Broken as of 2016-08-29
 def open_image_viewer(url, bdata):
 	global img
-	img = PhotoImage(data=bdata)
+	stream = BytesIO(bdata)
+	img = PhotoImage(data=stream.read())
```

Comment:
I had thought that `file` would have been the correct param...but it turned out to work well with data. 

If you don't like the read call inside the PhotoImage call this would work as well:


    stream = BytesIO(bdata)
    gif_image = stream.read()
    img = PhotoImage(data=gif_image)

Reply:
I'm fine with the direct call. This is in line with the overall coding style.